src/common/gui/SurgeGUIEditor.cpp

```diff
@@ -2478,6 +2478,21 @@ void SurgeGUIEditor::draw_infowindow(int ptag, CControl* control, bool modulate,
 
    if (buttons || forceMB)
    {
+      if (buttons || forceMB)
+      {
+         if (((CParameterTooltip*)infowindow)->isVisible())
+         {
+            ((CParameterTooltip*)infowindow)->Hide();
+            ((CParameterTooltip*)infowindow)->invalid();
+         }
+             
```

Comment:
You only need this block (if ->invisible) not the copy of the if and the inside. Your current code is structured as

```
if a
   if a
      if b
         x
      y
   else
      y
```

which is equivalent to

if a
  if b
     x
  y
```

see what I mean?